### PR TITLE
send invitation email automatically update

### PR DESCRIPTION
**Pull request by xiaoyaoxiangyun**

### CODE REVIEW

**Comment by xiaoyaoxiangyun** [comment]
an invitation email will be send along with the confirm email after confirming receiving the money. 
# The content of the invitation is for last year and need to be re-designed and updated

File: application/libraries/MY_Email.php [hunk 1]
```diff
@@ -78,4 +78,38 @@ public function send_fee_received_mail($mail, $school, $fee) {
         $message = $school . '，<br><br>贵校车协交来的' . $fee . '元参赛费用已经收到，感谢你们对北大赛的大力支持！如有任何问题，请直接与各地区负责联系。' . $id_message;
         $this->send_mail($mail, $subject, $message);
     }
+
+    /*
+     * Send invitation email. 
+     * $subject and $message needs to be updated.
+    */
```
Comment (MengchengJ) [inline application/libraries/MY_Email.php]: 星号要对齐

File: application/controllers/admin.php [hunk 2]
```diff
@@ -62,7 +62,13 @@ public function pay() {
                     $this->user->set_paid($data['id']);
                     $query= $this->user->get_user_by_id($data['id']);
                     $mail = $query['mail'];
+                    $association_name = $query['association_name'];
                     $this->email->send_fee_received_mail($mail, $data['school'], $data['bill']);
+
+                    // Send the invitation email. 
+                    $this->email->send_invitation_mail($mail, $data['school'], $association_name);
```
Comment (MengchengJ) [inline application/controllers/admin.php]: 把发送确认收款和邀请函合并到一个函数里，只发一封邮件吧

**Comment by JeldorPKU** [comment]
@MengchengJ 你看看不能rebase是不是因为有冲突，现在我这边看是可以rebase and merge的，如果你那不行的话你approve之后叫我

**Pull request closed without merging**